### ISSUE TITLE
Building Docker container failes

### ISSUE THREAD

**Issue by soyasis** [hiroyasuakada/mask-processing#2]
Hola!

There seems to be an issue with nvidia's [CUDA Linux Repository Key Rotation](https://forums.developer.nvidia.com/t/notice-cuda-linux-repository-key-rotation/212772).

There is a temporal fix to the dockerfile that seems to do the job as mentioned [here](https://github.com/NVIDIA/nvidia-container-toolkit/issues/257)

1. Add the following after `FROM nvidia/cuda:10.0-devel-ubuntu18.04`:
```
RUN rm /etc/apt/sources.list.d/cuda.list
RUN rm /etc/apt/sources.list.d/nvidia-ml.list
RUN apt-key del 7fa2af80
RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub
RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/7fa2af80.pub
```

2. remove the following:
```
RUN apt-get update && apt-get install -y --no-install-recommends \
    libcudnn7=$CUDNN_VERSION-1+cuda10.0 \
    libcudnn7-dev=$CUDNN_VERSION-1+cuda10.0 \
    && \
    apt-mark hold libcudnn7 && \
    rm -rf /var/lib/apt/lists/*
```

But unfortunately I cannot fully test the build as I still get some errors in my machine:
`executor failed running [/bin/sh -c mkdir /temporary_workspace/ && cd /temporary_workspace/ && git clone https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix.git && cd pytorch-CycleGAN-and-pix2pix && pip3 install -r requirements.txt]: exit code: 137`

Maybe you can try to build it and see if it works?
Also some insight into how and where you are running the app would be great to get it working. I am working on an image inpainting project and your human masking app is exactly what I need, but I'm ratehr new in the field and more familiar with Colab and Paperspace for running big models.

And also, I could not find the model you use `cascade_mask_rcnn_X_152_32x8d_FPN_IN5k_gn_dconv.pkl` in the detectron2 zoo, an id would be super helpful to easily search it! :)

**Comment by hiroyasuakada** [comment]
The original docker file worked on my environment but I don't quite remember it because it's a bit old work.

However, you can just install dependencies one by one.

For the app, you can simply follow the instructions on your terminal or something.

You can also download the pre-trained model of "Mask R-CNN X152" in the zoo.
